app/src/main/java/com/example/cm/ui/adapters/MeetupTapAdapter.java

```diff
@@ -0,0 +1,27 @@
+package com.example.cm.ui.adapters;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.adapter.FragmentStateAdapter;
+
+import com.example.cm.ui.dashboard.DashboardFragment;
+import com.example.cm.ui.meetup.MeetupListFragment;
+import com.example.cm.ui.meetup.MeetupNotificationsFragment;
+
+public class MeetupTapAdapter extends FragmentStateAdapter {
+
+    public MeetupTapAdapter(Fragment fragment) {
+        super(fragment);
+    }
+
+    @NonNull
+    @Override
+    public Fragment createFragment(int position) {
+        return position == 0 ? new MeetupListFragment(): new MeetupNotificationsFragment();
+    }
+
+    @Override
+    public int getItemCount() {
+        return 2;
```

Comment:
Is there a way to not hardcode this? Maybe store a list of Fragments to be used and add the size of the list instead.